packages/mdc-toolbar/mdc-toolbar.scss

```diff
@@ -21,12 +21,14 @@
 
 $mdc-toolbar-row-height: 64px;
 $mdc-toolbar-mobile-row-height: 56px;
+$mdc-toolbar-mobile-landscape-row-height: 48px;
 $mdc-toolbar-padding: 20px 28px;
-$mdc-toolbar-mobile-padding: 16px;
+$mdc-toobar-element-vertical-padding: 16px;
 
 $mdc-toolbar-ratio-to-extend-flexible: 4 !default;
 
-$mdc-toolbar-mobile-breakpoint: 599px;
+$mdc-toolbar-mobile-landscape-width-breakpoint: 960px;
+$mdc-toolbar-mobile-breakpoint: 600px;
```

Comment:
I agree we should strive for consistency, but in this case I think it makes more sense to just change the value getting assigned to the variable. It's much more readable that way.